newspaper/static/layout.css

```diff
@@ -25,22 +24,21 @@ body {
 
 .navbar-links {
     display: flex;
-    flex-direction: row;
```

Comment:
Yes - the default flex direction is row but let it be there just for precautionary sake